source/VehicleCommander/afmchemlight.json

```diff
@@ -1,124 +0,0 @@
-{
```

Comment:
If we are deleting these custom json files, we should also update(remove) those copy steps in the readme:
https://github.com/Esri/vehicle-commander-java/blob/master/README.md#configurecopy-the-required-files